java/kotlin-extractor/src/main/kotlin/KotlinUsesExtractor.kt

```diff
@@ -1008,12 +1015,13 @@ open class KotlinUsesExtractor(
 
     fun addJavaLoweringWildcards(t: IrType, addByDefault: Boolean, javaType: JavaType?): IrType =
         (t as? IrSimpleType)?.let {
+            val newAddByDefault = getWildcardSuppressionDirective(t)?.let { b -> !b } ?: addByDefault
```

Comment:
Is `?.let { b -> !b}` the same as `?.not()`? The latter feels easier to read.

Reply:
Done